src/components/views/Credentials/Credential.tsx

```diff
@@ -101,79 +101,81 @@ export const Credential: React.FunctionComponent<CredentialProps> = (props) => {
       {additionalVCData && (
         <>
           <Box
-            bg={expired || schemaMismatch ? colors.warning.light : colors.primary.disabled[1]}
+            bg={expired || schemaMismatch ? colors.warning.light : baseColors.blue}
```

Comment:
@nickreynolds I'm implementing the new credential designs they should be out next week. This blue isn't part of the design. It gets autogenerated so it's not actually one of our colors. I would remove all the updates on credential and the color theme.

Reply:
ok, I'll revert everything in Credential